app/bundles/InstallBundle/Configurator/Step/CheckStep.php

```diff
@@ -386,4 +393,27 @@ public function update(StepInterface $data)
 
         return $parameters;
     }
+
+    /**
+     * Takes the memory limit string form php.ini and returns numeric value in bytes.
+     *
+     * @param string $val
+     *
+     * @return integer
+     */
+    public function toBytes($val) {
+        $val = trim($val);
```

Comment:
I remember another issue/PR that fixed asset calculations when memory_limit = -1. I'm sure there aren't many out there with such a setting but just in case, maybe we can account for that here by checking if $val = -1 return PHP_MAX_INT